advance-building.rb

```diff
@@ -0,0 +1,27 @@
+def bubble_sort array
+  sorted_array = []
+  while (array.length != 0)
+    array.each_index do |index|
+      if (index != array.length-1 && array[index] > array[index+1])
+        array[index], array[index+1] = array[index+1], array[index]
+      end
+    end
+    sorted_array.insert(0, array.pop)
+  end
+  sorted_array
+end
```

Comment:
**[REQUIRED]**
Your method is making the given array empty. We need to fix your method to instead of returning a new array, sort the given array. 
`arr = [5,4,3,2,1]`
`p bubble_sort(arr)` => prints [1,2,3,4,5]
`p arr` => prints [ ]